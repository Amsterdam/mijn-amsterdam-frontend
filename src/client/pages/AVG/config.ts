import { AVGRequestFrontend } from '../../../server/services/avg/types';
import { dateSort } from '../../../universal/helpers/date';
import { LinkProps } from '../../../universal/types';
import {
  DisplayProps,
  WithDetailLinkComponent,
} from '../../components/Table/TableV2';

export const displayPropsAanvragen: DisplayProps<
  WithDetailLinkComponent<AVGRequestFrontend>
> = {
  detailLinkComponent: 'Nummer',
  ontvangstDatumFormatted: 'Ontvangen op',
  themas: 'Onderwerp(en)',
};

export const listPageParamKind = {
  inProgress: 'lopende-aanvragen',
  completed: 'afgehandelde-aanvragen',
};

const tableConfigBase = {
  sort: dateSort('dateRequest', 'desc'),
  displayProps: displayPropsAanvragen,
};

export const tableConfig = {
  [listPageParamKind.inProgress]: {
    title: 'Lopende aanvragen',
    filter: (avgVerzoek: AVGRequestFrontend) => !avgVerzoek.datumAfhandeling,
    ...tableConfigBase,
  },
  [listPageParamKind.completed]: {
    title: 'Afgehandelde aanvragen',
    filter: (avgVerzoek: AVGRequestFrontend) => avgVerzoek.datumAfhandeling,
    ...tableConfigBase,
  },
};

export const linkListItems: LinkProps[] = [
  {
    to: 'https://www.amsterdam.nl/privacy/loket/',
    title: 'Loket persoonsgegevens gemeente Amsterdam',
  },
];
