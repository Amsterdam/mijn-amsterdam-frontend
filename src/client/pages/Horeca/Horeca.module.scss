@import '../../styles/_global.scss';

.Horeca {
  @include select(PageContent) {
    padding-bottom: $page-content-bottom-space;
  }
}
