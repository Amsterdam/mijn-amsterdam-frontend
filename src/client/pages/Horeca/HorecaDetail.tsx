import { useHorecaThemaData } from './useHorecaThemaData.hook';
import {
  DecosZaakExploitatieHorecabedrijf,
  HorecaVergunningFrontend,
} from '../../../server/services/horeca/config-and-types';
import { VergunningFrontend } from '../../../server/services/vergunningen/config-and-types';
import { Datalist } from '../../components/Datalist/Datalist';
import DocumentListV2 from '../../components/DocumentList/DocumentListV2';
import { PageContentCell } from '../../components/Page/Page';
import ThemaDetailPagina from '../ThemaPagina/ThemaDetailPagina';
import { getRows } from '../Vergunningen/detail-page-content/fields-config';
import { useVergunningenDetailData } from '../Vergunningen/useVergunningenDetailData.hook';

type ExploitatieHorecaBedrijfProps = {
  vergunning: VergunningFrontend<DecosZaakExploitatieHorecabedrijf>;
};

function ExploitatieHorecaBedrijf({
  vergunning,
}: ExploitatieHorecaBedrijfProps) {
  const rows = getRows(vergunning, [
    'identifier',
    'location',
    {
      dateStart: () => ({
        label: 'Begindatum',
        content: vergunning.dateStartFormatted,
        isVisible: vergunning.processed && vergunning.decision === 'Verleend',
      }),
    },
    'decision',
  ]);

  return <Datalist rows={rows} />;
}

export function HorecaDetailPagina() {
  const { vergunningen, isLoading, isError, themaPaginaBreadcrumb } =
    useHorecaThemaData();
  const { vergunning, title, documents } =
    useVergunningenDetailData<HorecaVergunningFrontend>(vergunningen);

  return (
    <ThemaDetailPagina<HorecaVergunningFrontend>
      title={title}
      zaak={vergunning}
      isError={isError}
      isLoading={isLoading}
      pageContentMain={
        vergunning && (
          <>
            <PageContentCell>
              <ExploitatieHorecaBedrijf vergunning={vergunning} />
            </PageContentCell>
            {!!documents.length && (
              <PageContentCell spanWide={8}>
                <Datalist
                  rows={[
                    {
                      label: 'Documenten',
                      content: (
                        <DocumentListV2
                          documents={documents}
                          columns={['', '']}
                        />
                      ),
                    },
                  ]}
                />
              </PageContentCell>
            )}
          </>
        )
      }
      breadcrumbs={[themaPaginaBreadcrumb]}
    />
  );
}
