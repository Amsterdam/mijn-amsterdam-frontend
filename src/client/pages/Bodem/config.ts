import { generatePath } from 'react-router-dom';

import { LoodMetingFrontend } from '../../../server/services/bodem/types';
import { AppRoutes } from '../../../universal/config/routes';
import { dateSort } from '../../../universal/helpers/date';
import { LinkProps } from '../../../universal/types';

const displayPropsAanvragen = {
  detailLinkComponent: 'Adres',
  datumAanvraagFormatted: 'Aangevraagd',
  status: 'Status',
};

const listPageParamKind = {
  inProgress: 'lopende-aanvragen',
  completed: 'afgehandelde-aanvragen',
};

const tableConfigBase = {
  sort: dateSort('dateRequest', 'desc'),
  displayProps: displayPropsAanvragen,
};

export const tableConfig = {
  [listPageParamKind.inProgress]: {
    ...tableConfigBase,
    title: 'Lopende aanvragen',
    filter: (bodemAanvraag: LoodMetingFrontend) => !bodemAanvraag.processed,
    listPageRoute: generatePath(AppRoutes['BODEM/LIST'], {
      kind: listPageParamKind.inProgress,
    }),
  },
  [listPageParamKind.completed]: {
    ...tableConfigBase,
    title: 'Afgehandelde aanvragen',
    filter: (bodemAanvraag: LoodMetingFrontend) => bodemAanvraag.processed,
    listPageRoute: generatePath(AppRoutes['BODEM/LIST'], {
      kind: listPageParamKind.completed,
    }),
  },
} as const;

export const linkListItems: LinkProps[] = [
  {
    title: 'Meer informatie over lood in de bodem.',
    to: 'https://www.amsterdam.nl/wonen-leefomgeving/bodem/lood-grond/',
  },
] as const;
