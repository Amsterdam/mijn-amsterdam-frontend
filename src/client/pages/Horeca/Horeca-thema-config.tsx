import { generatePath } from 'react-router-dom';

import { ExploitatieHorecabedrijf } from '../../../server/services/vergunningen/vergunningen';
import { ExploitatieHorecabedrijf as ExploitatieHorecabedrijfV2 } from '../../../server/services/vergunningen-v2/config-and-types';
import { AppRoutes } from '../../../universal/config/routes';
import { LinkProps } from '../../../universal/types/App.types';
import {
  DisplayProps,
  WithDetailLinkComponent,
} from '../../components/Table/TableV2';

export type HorecaVergunning =
  | ExploitatieHorecabedrijf
  | ExploitatieHorecabedrijfV2;

const MAX_TABLE_ROWS_ON_THEMA_PAGINA_LOPEND = 5;

export const LinkListItems: LinkProps[] = [
  // {
  //   title: '',
  //   to: '',
  // },
];

const displayPropsHorecaHuidig: DisplayProps<
  WithDetailLinkComponent<HorecaVergunning>
> = {
  detailLinkComponent: 'Zaaknummer',
  title: 'Soort vergunning',
  dateRequestFormatted: 'Aangevraagd',
};

const displayPropsHorecaEerder: DisplayProps<
  WithDetailLinkComponent<HorecaVergunning>
> = {
  detailLinkComponent: 'Zaaknummer',
  title: 'Soort vergunning',
  decision: 'Resultaat',
};

export const routes = {
  listPage: AppRoutes['HORECA/LIST'],
  detailPageVergunning: AppRoutes['HORECA/DETAIL'],
  themaPage: AppRoutes.HORECA,
} as const;

export const listPageParamKind = {
  huidig: 'huidige-aanvragen-en-vergunningen',
  eerder: 'eerdere-aanvragen-en-vergunningen',
} as const;

export type ListPageParamKey = keyof typeof listPageParamKind;
export type ListPageParamKind = (typeof listPageParamKind)[ListPageParamKey];

export const horecaTableConfig = {
  [listPageParamKind.huidig]: {
    title: 'Huidige aanvragen en vergunningen',
    filter: (v: HorecaVergunning) => !v.processed,
    textNoContent: 'U heeft geen horeca vergunningen of aanvragen.',
    displayProps: displayPropsHorecaHuidig,
    maxItems: MAX_TABLE_ROWS_ON_THEMA_PAGINA_LOPEND,
    listPageRoute: generatePath(routes.listPage, {
      kind: listPageParamKind.huidig,
    }),
  },
  [listPageParamKind.eerder]: {
    title: 'Eerdere aanvragen en vergunningen',
    filter: (v: HorecaVergunning) => v.processed,
    textNoContent: 'U heeft geen eerdere horeca vergunningen of aanvragen.',
    displayProps: displayPropsHorecaEerder,
    maxItems: MAX_TABLE_ROWS_ON_THEMA_PAGINA_LOPEND,
    listPageRoute: generatePath(routes.listPage, {
      kind: listPageParamKind.eerder,
    }),
  },
} as const;
