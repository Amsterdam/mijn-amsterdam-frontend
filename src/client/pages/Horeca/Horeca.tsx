import { AppRoutes, ChapterTitles } from '../../../universal/config';
import {
  defaultDateFormat,
  isError,
  isLoading,
} from '../../../universal/helpers';
import {
  addTitleLinkComponent,
  ErrorAlert,
  ChapterIcon,
  OverviewPage,
  PageContent,
  PageHeading,
  SectionCollapsible,
  Table,
} from '../../components';
import { useAppStateGetter } from '../../hooks';
import styles from './Horeca.module.scss';

const DISPLAY_PROPS_HORECA = {
  identifier: 'Kenmerk',
  title: 'Soort vergunning',
  dateRequest: 'Aangevraagd',
};

const DISPLAY_PROPS_HORECA_EERDER = {
  identifier: 'Kenmerk',
  title: 'Soort vergunning',
  decision: 'Resultaat',
};

export const HORECA_PAGE_SIZE = 20;

export default function Horeca() {
  const { HORECA } = useAppStateGetter();

  const vergunningen =
    HORECA.content?.map((v) => {
      return {
        ...v,
        dateRequest: defaultDateFormat(v.dateRequest),
      };
    }) || [];

  const items = addTitleLinkComponent(vergunningen, 'identifier');

  const lopendeVergunningen = items.filter((v) => !v.processed);
  const afgerondeVergunningen = items.filter((v) => v.processed);

  return (
    <OverviewPage className={styles.Horeca}>
      <PageHeading
        backLink={{
          to: AppRoutes.HOME,
          title: 'Home',
        }}
        isLoading={isLoading(HORECA)}
        icon={<ChapterIcon />}
      >
        {ChapterTitles.HORECA}
      </PageHeading>
      <PageContent>
        <p>
          Hier ziet u een overzicht van uw aanvragen voor Horeca en ontheffingen
          bij gemeente Amsterdam.
        </p>
        {isError(HORECA) && (
          <ErrorAlert>
            We kunnen op dit moment geen vergunningen tonen.
          </ErrorAlert>
        )}
      </PageContent>
      <SectionCollapsible
        id="SectionCollapsible-complaints"
        title="Lopende aanvragen"
        noItemsMessage="U heeft geen horeca vergunningen in aanvraag."
        startCollapsed={false}
        hasItems={!!lopendeVergunningen?.length}
        isLoading={isLoading(HORECA)}
        className=""
      >
        <Table
          items={lopendeVergunningen}
          displayProps={DISPLAY_PROPS_HORECA}
          titleKey="identifier"
        />
      </SectionCollapsible>
      <SectionCollapsible
        id="SectionCollapsible-complaints"
        title="Eerdere aanvragen"
        noItemsMessage="U heeft nog geen toegekende horeca vergunningen."
        startCollapsed={false}
        hasItems={!!afgerondeVergunningen?.length}
        isLoading={isLoading(HORECA)}
        className=""
      >
        <Table
          items={afgerondeVergunningen}
          displayProps={DISPLAY_PROPS_HORECA_EERDER}
          titleKey="identifier"
        />
      </SectionCollapsible>
    </OverviewPage>
  );
}
