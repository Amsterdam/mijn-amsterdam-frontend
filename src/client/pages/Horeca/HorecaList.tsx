import { useParams } from 'react-router-dom';

import { ListPageParamKind } from './Horeca-thema-config';
import { useHorecaThemaData } from './useHorecaThemaData.hook';
import { ListPagePaginated } from '../../components/ListPagePaginated/ListPagePaginated';

export function HorecaLijstPagina() {
  const { kind } = useParams<{ kind: ListPageParamKind }>();
  const {
    vergunningen,
    tableConfig,
    routes,
    isLoading,
    isError,
    themaPaginaBreadcrumb,
  } = useHorecaThemaData();
  const listPageTableConfig = tableConfig[kind];

  return (
    <ListPagePaginated
      items={vergunningen.filter(listPageTableConfig.filter)}
      title={listPageTableConfig.title}
      appRoute={routes.listPage}
      appRouteParams={{ kind }}
      breadcrumbs={[themaPaginaBreadcrumb]}
      displayProps={listPageTableConfig.displayProps}
      isLoading={isLoading}
      isError={isError}
    />
  );
}
