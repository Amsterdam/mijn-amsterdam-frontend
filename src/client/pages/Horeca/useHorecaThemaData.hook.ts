import { isError } from 'lodash';

import { tableConfig, LinkListItems, routes } from './Horeca-thema-config';
import { HorecaVergunning } from '../../../server/services/horeca/config-and-types';
import { VergunningFrontend } from '../../../server/services/vergunningen/config-and-types';
import { isLoading } from '../../../universal/helpers/api';
import { addLinkElementToProperty } from '../../components/Table/TableV2';
import { ThemaTitles } from '../../config/thema';
import { useAppStateGetter } from '../../hooks/useAppState';

export function useHorecaThemaData() {
  const { HORECA } = useAppStateGetter();

  const vergunningen = addLinkElementToProperty<
    VergunningFrontend<HorecaVergunning>
  >(HORECA.content ?? [], 'identifier', true);

  return {
    vergunningen,
    isLoading: isLoading(HORECA),
    isError: isError(HORECA),
    linkListItems: LinkListItems,
    routes,
    tableConfig,
    themaTitle: ThemaTitles.HORECA,
  };
}
