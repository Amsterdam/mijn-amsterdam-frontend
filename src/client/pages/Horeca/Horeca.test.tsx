import { render } from '@testing-library/react';
import { generatePath } from 'react-router-dom';
import { MutableSnapshot } from 'recoil';
import {
  addLinks,
  transformVergunningenData,
  VergunningenSourceData,
} from '../../../server/services';
import vergunningenData from '../../../server/mock-data/json/vergunningen.json';
import { AppRoutes } from '../../../universal/config';
import { horecaOptions } from '../../../server/services/horeca';
import { appStateAtom } from '../../hooks';
import MockApp from '../MockApp';
import Horeca from './Horeca';

const testState: any = {
  HORECA: {
    status: 'OK',
    content: addLinks(
      transformVergunningenData(
        vergunningenData as VergunningenSourceData
      ).filter(horecaOptions.filter),
      AppRoutes['HORECA/DETAIL']
    ),
  },
};

function initializeState(snapshot: MutableSnapshot) {
  snapshot.set(appStateAtom, testState);
}

describe('<Horeca />', () => {
  const routeEntry = generatePath(AppRoutes.HORECA);
  const routePath = AppRoutes.HORECA;
  const Component = () => (
    <MockApp
      routeEntry={routeEntry}
      routePath={routePath}
      component={Horeca}
      initializeState={initializeState}
    />
  );

  it('Matches the Full Page snapshot', () => {
    const { asFragment } = render(<Component />);
    expect(asFragment()).toMatchSnapshot();
  });
});
