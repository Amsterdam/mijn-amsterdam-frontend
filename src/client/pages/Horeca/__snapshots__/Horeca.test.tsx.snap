// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Horeca /> > Matches the Full Page snapshot 1`] = `
<DocumentFragment>
  <main
    class="_Page_47d7c0 _OverviewPage_47d7c0 _Horeca_41591c"
  >
    <header
      class="_PageHeading_3b337d _hasIcon_3b337d _hasBackLink_3b337d"
    >
      <span
        aria-hidden="true"
        class="_Icon_3b337d"
      >
        <svg
          aria-label="Horeca"
          class="_ThemaIcon_2fc54d"
          fill="black"
          height="32"
          viewBox="0 0 25 32"
          width="25"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M6 32V17C3 17 0 13.8962 0 11V0H3V11H6V0H9V11H11.94L12 0H15V11C15 14 11.94 17 9 17V32H6ZM22 32V19H18V7C18 4.89333 18.64 3.2 19.92 1.92C21.2 0.64 22.8933 0 25 0V32H22Z"
          />
        </svg>
      </span>
      <div
        class="_HeadingInner_3b337d"
      >
        <a
          class="ams-link ams-link--standalone _BackLink_3b337d"
          href="/"
        >
          <span
            class="ams-icon ams-icon--size-5"
          >
            <svg
              aria-hidden="true"
              focusable="false"
              viewBox="0 0 32 32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="m22.857 32-16-16 16-16 2.9 2.91L12.677 16l13.08 13.09z"
                fill-rule="evenodd"
              />
            </svg>
          </span>
          <span>
            Terug
          </span>
        </a>
        <h2
          class="ams-heading ams-heading--level-1"
        >
          Horeca
        </h2>
      </div>
    </header>
    <div
      class="_PageContent_47d7c0"
    >
      <p>
        Hier ziet u een overzicht van uw aanvragen voor Horeca en ontheffingen bij gemeente Amsterdam.
      </p>
    </div>
    <section
      class="_SectionCollapsible_b93dc7"
    >
      <h3
        class="ams-heading ams-heading--level-3 _Title_b93dc7 _TitleWithItems_b93dc7"
      >
        <button
          aria-expanded="true"
          class="_TitleToggle_b93dc7"
        >
          <svg
            aria-hidden="true"
            aria-label="chevron-rechts-icoon"
            class="_CaretIcon_b93dc7"
            focusable="false"
            height="32px"
            viewBox="0 0 32 32"
            width="32px"
            xmlns="http://www.w3.org/2000/svg"
            xmlns:xlink="http://www.w3.org/1999/xlink"
          >
            <polygon
              points="9.75688579 32 6.85714286 29.0899743 19.9368344 16 6.85714286 2.91002571 9.75688579 0 25.7568858 16"
            />
          </svg>
           Lopende aanvragen
        </button>
      </h3>
      <div
        aria-hidden="false"
        class="_Panel_b93dc7"
        style="height: 0px;"
      >
        <div
          class="_PanelInner_b93dc7"
        >
          <table
            class="_Table_a66dcc"
          >
            <thead>
              <tr
                class="_TableRow_a66dcc"
              >
                <th
                  class="_DisplayProp_a66dcc"
                >
                  Kenmerk
                </th>
                <th
                  class="_DisplayProp_a66dcc"
                >
                  Soort vergunning
                </th>
                <th
                  class="_DisplayProp_a66dcc"
                >
                  Aangevraagd
                </th>
              </tr>
            </thead>
            <tbody>
              <tr
                class="_TableRow_a66dcc"
              >
                <td
                  class="_DisplayPropTitle_a66dcc"
                >
                  <a
                    class="_Button_c3ab45 _Button__plain_c3ab45 _Button__lean_c3ab45 _Linkd_c3ab45"
                    href="/horeca/horeca-vergunning-exploitatie-horecabedrijf/334568232"
                  >
                    <svg
                      aria-hidden="true"
                      aria-label="chevron-rechts-icoon"
                      class="_Icon_c3ab45 _Icon__left_c3ab45"
                      fill="#000"
                      focusable="false"
                      height="14px"
                      viewBox="0 0 32 32"
                      width="14px"
                      xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink"
                    >
                      <polygon
                        points="9.75688579 32 6.85714286 29.0899743 19.9368344 16 6.85714286 2.91002571 9.75688579 0 25.7568858 16"
                      />
                    </svg>
                    Z/23/1808827
                  </a>
                </td>
                <td
                  class="_DisplayProp_a66dcc"
                >
                  <span
                    class="_DisplayPropLabel_a66dcc"
                  >
                    Soort vergunning:
                  </span>
                  <span>
                    Horeca vergunning exploitatie Horecabedrijf
                  </span>
                </td>
                <td
                  class="_DisplayProp_a66dcc"
                >
                  <span
                    class="_DisplayPropLabel_a66dcc"
                  >
                    Aangevraagd:
                  </span>
                  <span>
                    20 november 2022
                  </span>
                </td>
              </tr>
              <tr
                class="_TableRow_a66dcc"
              >
                <td
                  class="_DisplayPropTitle_a66dcc"
                >
                  <a
                    class="_Button_c3ab45 _Button__plain_c3ab45 _Button__lean_c3ab45 _Linkd_c3ab45"
                    href="/horeca/horeca-vergunning-exploitatie-horecabedrijf/3209922248"
                  >
                    <svg
                      aria-hidden="true"
                      aria-label="chevron-rechts-icoon"
                      class="_Icon_c3ab45 _Icon__left_c3ab45"
                      fill="#000"
                      focusable="false"
                      height="14px"
                      viewBox="0 0 32 32"
                      width="14px"
                      xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink"
                    >
                      <polygon
                        points="9.75688579 32 6.85714286 29.0899743 19.9368344 16 6.85714286 2.91002571 9.75688579 0 25.7568858 16"
                      />
                    </svg>
                    Z/23/1808826
                  </a>
                </td>
                <td
                  class="_DisplayProp_a66dcc"
                >
                  <span
                    class="_DisplayPropLabel_a66dcc"
                  >
                    Soort vergunning:
                  </span>
                  <span>
                    Horeca vergunning exploitatie Horecabedrijf
                  </span>
                </td>
                <td
                  class="_DisplayProp_a66dcc"
                >
                  <span
                    class="_DisplayPropLabel_a66dcc"
                  >
                    Aangevraagd:
                  </span>
                  <span>
                    20 oktober 2022
                  </span>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </section>
    <section
      class="_SectionCollapsible_b93dc7"
    >
      <h3
        class="ams-heading ams-heading--level-3 _Title_b93dc7 _TitleWithItems_b93dc7"
      >
        <button
          aria-expanded="true"
          class="_TitleToggle_b93dc7"
        >
          <svg
            aria-hidden="true"
            aria-label="chevron-rechts-icoon"
            class="_CaretIcon_b93dc7"
            focusable="false"
            height="32px"
            viewBox="0 0 32 32"
            width="32px"
            xmlns="http://www.w3.org/2000/svg"
            xmlns:xlink="http://www.w3.org/1999/xlink"
          >
            <polygon
              points="9.75688579 32 6.85714286 29.0899743 19.9368344 16 6.85714286 2.91002571 9.75688579 0 25.7568858 16"
            />
          </svg>
           Eerdere aanvragen
        </button>
      </h3>
      <div
        aria-hidden="false"
        class="_Panel_b93dc7"
        style="height: 0px;"
      >
        <div
          class="_PanelInner_b93dc7"
        >
          <table
            class="_Table_a66dcc"
          >
            <thead>
              <tr
                class="_TableRow_a66dcc"
              >
                <th
                  class="_DisplayProp_a66dcc"
                >
                  Kenmerk
                </th>
                <th
                  class="_DisplayProp_a66dcc"
                >
                  Soort vergunning
                </th>
                <th
                  class="_DisplayProp_a66dcc"
                >
                  Resultaat
                </th>
              </tr>
            </thead>
            <tbody>
              <tr
                class="_TableRow_a66dcc"
              >
                <td
                  class="_DisplayPropTitle_a66dcc"
                >
                  <a
                    class="_Button_c3ab45 _Button__plain_c3ab45 _Button__lean_c3ab45 _Linkd_c3ab45"
                    href="/horeca/horeca-vergunning-exploitatie-horecabedrijf/1234418712"
                  >
                    <svg
                      aria-hidden="true"
                      aria-label="chevron-rechts-icoon"
                      class="_Icon_c3ab45 _Icon__left_c3ab45"
                      fill="#000"
                      focusable="false"
                      height="14px"
                      viewBox="0 0 32 32"
                      width="14px"
                      xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink"
                    >
                      <polygon
                        points="9.75688579 32 6.85714286 29.0899743 19.9368344 16 6.85714286 2.91002571 9.75688579 0 25.7568858 16"
                      />
                    </svg>
                    Z/23/1984708
                  </a>
                </td>
                <td
                  class="_DisplayProp_a66dcc"
                >
                  <span
                    class="_DisplayPropLabel_a66dcc"
                  >
                    Soort vergunning:
                  </span>
                  <span>
                    Horeca vergunning exploitatie Horecabedrijf
                  </span>
                </td>
                <td
                  class="_DisplayProp_a66dcc"
                >
                  <span
                    class="_DisplayPropLabel_a66dcc"
                  >
                    Resultaat:
                  </span>
                  <span>
                    Verleend
                  </span>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </section>
  </main>
</DocumentFragment>
`;
